### DISCUSSION TITLE
Keyboard backlight action - set to the most recently used preset

### DISCUSSION THREAD

**Discussion by 1aTa** [BartoszCichecki/LenovoLegionToolkit#1017] [Ideas]
Provide a keyboard backlight action which sets the backlight to the most recently used preset.

**Comment by BartoszCichecki** [comment]
"Everything is possible", however due to how LLT was architected, this would be quite hard to implement into current architecture. I don't think that this will be implemented by me anytime soon. However if someone wants to have a go, by all means try.
